tests/integration/utils.py

```diff
@@ -6,7 +6,7 @@
 import shutil
 import string
 from dataclasses import dataclass, field
-from pathlib import Path
+from pathlib import Path, PurePath
```

Comment:
IMO we can use just Path everywhere

PurePath is useful when you want to make sure you're not doing any I/O (reading, writing, checking existence...) but we usually do want to do those things